modules/webbrowser/webhelper/CMakeLists.txt

```diff
@@ -23,12 +23,13 @@ set(WEBBROWSER_HELPER_SOURCES
     ${WEBBROWSER_RESOURCES}
 )
 
-# place helper in separate executable. CEF uses "<app name> + Helper" by default
+# place helper in separate executable. 
 set(CEF_HELPER_TARGET "cef_inviwo_helper" CACHE INTERNAL "CEF_HELPER_TARGET")
-set(CEF_HELPER_OUTPUT_NAME "${IVW_APP_INSTALL_NAME} Helper")
+
 
 if (NOT OS_MACOSX)
     add_executable(${CEF_HELPER_TARGET} WIN32 MACOSX_BUNDLE ${WEBBROWSER_HELPER_SOURCES})
```

Comment:
You you need to create a app bundle would it not be easier just do do the plain executable? just skip MACOSX_BUNDLE

Reply:
Yes, but I vaguely recall that I tried once and that CEF relies on it being a bundle, but I'm not sure. Probably best to follow their example to keep it consistent with their code.